python/tvm/relay/frontend/onnx.py

```diff
@@ -945,7 +945,7 @@ def _impl_v1(cls, inputs, attr, params):
         else:
             attr["layout"] = onnx_default_layout(dims=(len(input_shape) - 2), op_name="LpPool")
 
-        p = _expr.const(attr["p"], dtype)
+        p = _expr.const(attr.get("p", 2), dtype)
         reci_p = _expr.const(1.0 / attr["p"], dtype)
```

Comment:
oh I didn't spot it, please also add default value for this

Reply:
I see.